purchase_cancel_qty/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Purchase Cancel Qty
+===================
+
+Allow purchase users to define cancelled quantity on purchase order lines.
+* Quantity field becomes readonly when a PO is confirmed and its value is stored in an other field
+* A new field named 'Cancelled quantity' will update the quantity based on the initially ordered quantity
+
+Installation
+============
+
+Just install it.
+
+Configuration
+=============
+
+No configuration needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go on the PO form view
+* Just update the cancelled quantity field
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Benjamin Willig <benjamin.willig@acsone.eu>
+
+Funders
+-------
+
+The development of this module has been financially supported by:
+
+* ACSONE SA/NV (https://acsone.eu)
```

Comment:
Please remove the Funders section, this work has been financed by our client.

Reply:
Done.